apps/meteor/client/views/account/preferences/PreferencesNotificationsSection.tsx

```diff
@@ -34,26 +35,40 @@ const PreferencesNotificationsSection = ({ onChange, commitRef, ...props }: Form
 		'Accounts_Default_User_Preferences_pushNotifications',
 	) as keyof typeof notificationOptionsLabelMap;
 	const canChangeEmailNotification = useSetting('Accounts_AllowEmailNotifications');
+	const showNewLoginEmailPreference = useSetting('Device_Management_Allow_Login_Email_preference');
```

Comment:
```suggestion
	const showNewLoginEmailPreference = useSetting('Device_Management_Enable_Login_Emails') && useSetting('Device_Management_Allow_Login_Email_preference');
```
Not back-end related, but I was wondering if maybe we should show this preference only when email notifications are enabled 🤔 so that the user can't enable/disable it if they're not gonna receive an email anyways

Reply:
Makes sense. Thanks!